### PR TITLE
RD-24 Working with repositories

### PR DESCRIPTION
**Pull request by thepatrykk**
2 cases to consider:
- Creating the new branch and push it to GitHub
- Push code to existing repository

Feel free to comment.

https://aurity.atlassian.net/browse/RD-24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +53,34 @@ There are many (too many!) ways to write React components so to make sure we all
 |       ├── component-name.scss
 ```
 
+## Working with repositories
```
Comment (PeterKow) [inline README.md]: can we move this to different file => `docs/gitflow` so this readme it;s not too long

**Review by PeterKow — CHANGES_REQUESTED** [review]
can you add info how good PR looks like:

Clean, small pieces of code
eslint + tests before final commit
check your PR and compare with ticket requirements before releasing to others
add link to jira ticket in PR notes
take a print screen or use Licecap or Gayzao to record a screen

**Comment by PeterKow** [comment]
can you also add info about a review process:
-> create PR 
-> check your PR
-> share on slack and put into "review" column in Jira
->  PR should be reviewed by at lease two people and  👍 x2  when all comments are solved
-> after review ticket can be QA (move in Jira)

-> PR should be created again base branch `develop`

**Comment by thepatrykk** [comment]
The changes:
Main README

188bffbb9b8222aec74eb41fcd44edf6

docs/gitflow/README
6723199e5b79627eb4e8b9a29d30a8b1

**Comment by 8Pablo** [comment]
Nice one Patryk.

**Comment by Bran1304** [comment]
Looking good

File: docs/gitflow/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+If you've copied a project repository, cloned it down to your local system, and added a Git remote to your clone that point to the original repository on GitHub, now you're ready to make some changes in code. Far as you will be making changes to your local Git repository and collaborate with others on the same repo, you should use a branch.
+
+The flow looks like this:
+1. Create and checkout a new branch.
+2. Make changes.
+3. Commit your changes to the branch.
+4. Push the changes to GitHub.
+
+To create a new branch and check it out (telling Git you will be making changes to the branch), use this command:
+### `git checkout -b <name_of_your_new_branch>`
```
Comment (PeterKow) [inline docs/gitflow/README.md]: ` branch name should be `feature/name_<jira-ticket>`

File: docs/gitflow/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+* [Working with repositories](#working-with-repositories)
+* [Pull Request](#pull-request)
+* [Review Process](#review-process)
+
+## Working with repositories
+If you've copied a project repository, cloned it down to your local system, and added a Git remote to your clone that point to the original repository on GitHub, now you're ready to make some changes in code. Far as you will be making changes to your local Git repository and collaborate with others on the same repo, you should use a branch.
```
Comment (PeterKow) [inline docs/gitflow/README.md]: we can mention that SourceTree is a good tool to manage your repos

File: docs/gitflow/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+Use the base branch drop-down menu to select the branch you'd like to merge your changes into:
+![select-branch](https://i.gyazo.com/9e44805dcb50ba138c7d655c17ae8ee9.png)
+
+Enter a title and description of the pull request.
+
+Effective Pull Request looks like:
+1. Make it small and clean piece of code.
+2. Make sure all tests pass - ESLINT and tests before final commit.
+3. Checked and compare with ticket requirements before releasing to others.
+4. In description adding a link to Jira ticket.
```
Comment (PeterKow) [inline docs/gitflow/README.md]: In description adding a link to Jira ticket and notes if any + screenshot (I think I've sent you also tools for making gifs)

**Review by PeterKow — CHANGES_REQUESTED** [review]
Last changes and ready to go

**Comment by thepatrykk** [comment]
SourceTree, Gyazo

**Review by PeterKow — APPROVED** [review]

**Pull request merged**